pkg/cloudprovider/servicelb.go

```diff
@@ -55,7 +55,7 @@ const (
 )
 
 var (
-	DefaultLBImage = "rancher/klipper-lb:v0.4.9"
+	DefaultLBImage = "registry.suse.com/bci/bci-busybox:latest"
```

Comment:
Don't use the `latest` tag, and this needs to come from something in the `rancher/mirrored-` images alongside everything else.

Reply:
no, but not using latest is a best practice, and moving off docker hub to registry.suse.com would be a bigger deal as existing users may expect all of our images to be under the `docker.io/rancher` namespace, and we'd want to coordinate that change much further out.